### ISSUE TITLE
seqkit amplicon only keeps one amplicon 

### ISSUE THREAD

**Issue by mlosilla** [shenwei356/seqkit#191]
Hi,

In version 0.15.0 seqkit amplicon only keeps one amplicon (the largest) per primer pair. I don't always care about the largest amplicon. 

Would it be possible to implement the feature of keeping all valid amplicons? A bed file with the start and end bases (columns 2 and 3) of all valid pairs would be fantastic. That would permit me to inspect the amplicons and keep the one I want (in my case, the smallest amplicon over a certain threshold).

Thanks

**Comment by shenwei356** [comment]
Right, PCR could produce all combinations of the forward and backward primers. We should output them too.

**Comment by mlosilla** [comment]
yeah exactly. 

In my case, the positions in the bed file would be enough. I would choose the amplicon I want and use the positions as trimming points for my fastq reads--but the sequences of the PCR products may be useful for other users.

Thank you for considering it!

**Comment by shenwei356** [comment]
> In my case, the positions in the bed file would be enough.

Oh, you can use `seqkit loate` first, which outputs BED format.

**Comment by zjhzxjm** [comment]
期待这个功能早日上线
